src/api/BioLink.js

```diff
@@ -213,10 +213,39 @@ async function getCountsForNode(nodeId, nodeType) {
 
     return associationsResultMap;
   }
-  console.log('getCountsForNode', nodeId, nodeType, 'NO ASSOCIATIONS KNOWN');
+
+  // console.log('getCountsForNode', nodeId, nodeType, 'NO ASSOCIATIONS KNOWN');
   return null;
 }
 
+
+async function getCountsForNodeNew(nodeId, nodeType) {
+  const bioentityUrl = `${biolink}bioentity/${nodeType}/${nodeId}`;
+  console.log('getCountsForNodeNew', nodeId, nodeType);
+  const bioentityParams = {
+    fetch_objects: false,
+    unselect_evidence: true,
+    exclude_automatic_assertions: false,
+    use_compact_associations: true,
+    get_association_counts: true,
+    rows: 1
+  };
+  const bioentityResp = await axios.get(bioentityUrl, { params: bioentityParams });
+  const bioentityResponseData = bioentityResp.data;
+  console.log(bioentityResp.request.responseURL);
```

Comment:
remove loggin unless not needed

Reply:
This logging may be necessary if BL's `get_association_counts` gets fixed and I need to test the new version of getCounts above. Once I'm confident the BL endpoint works, I'll remove the old count code and all the logging. Meanwhile, I'm leaving it in.

Reply:
https://github.com/monarch-initiative/monarch-ui/pull/40/files/e965086cb5924a2386508ae246f5c6757874ddac#diff-ec1bbcbf802856ad9001d1b8c3b89ce3R296